blueocean-core-js/package.json

```diff
@@ -59,8 +55,8 @@
     "chai": "3.5.0",
     "del": "2.2.2",
     "enzyme": "2.4.1",
-    "eslint-plugin-react": "6.3.0",
-    "flow-bin": "^0.34.0",
+    "eslint-plugin-react": "4.3.0",
```

Comment:
The same dep was defined in both "dependencies" (as 4.3.0) and "devDependencies" (as 6.3.0). So 4.3.0 was the effective version used by the project. I removed it from "dependencies" (should not have been added there in the first place) so this change here is just adjusting the version number to what it really was before. We are using 4.3.0 in all other projects, and if I recall there are some peerDeps conflicts with js-builder when trying to use the newer version anyhow, so I think 4.3.0 is the right choice here.